### ISSUE TITLE
Need info about how to use/try the beta and what limitations there are?

### ISSUE THREAD

**Issue by ghost** [fsprojects/fsharp-dnx#23]
I am eager for DNX support so I can call into a F# project from a ASP.NET 5 DNX project. I would like to try the beta but lack instructions about how to use it? E.x. where should I install it (VS, solution etc?) What can it do and what can it not do ?

**Comment by Alxandr** [comment]
Depending on what you mean by "the beta", there is no installing. It's just a dependency. See the sample projects in this repository.

**Comment by ghost** [comment]
I want to create a F# dnx library that a C# dnx ASP.NET 5 project can depend on. Since there is no F# dnx supprot in VS, do I then need to create a C# dnx library and then change it afterwards to look like one of your samples OR ?

**Comment by Alxandr** [comment]
In general. VS tooling for F# DNX libraries are non-existent. You can trick it, but it's a bit of a pita. Take a look at https://github.com/YoloDev/Nini for sample on just that. What I did was create a normal F# project, add the `project.json`, and then manually add the references that's needed.

**Comment by ghost** [comment]
Does this only work if your F# project is the "master project" that only draw in dependencies? I.e. would there be problems if another project (e.x. a C# ASP.NET DNX) project is that "master project" and has a dependency on the F# project? Also, how did you create your .xproj files in your samples - since I get .fsproj files if I create a F# project.

**Comment by Alxandr** [comment]
The samples (in this repo) doesn't "work" in VS.

**Comment by ghost** [comment]
Surprise - Useful to know. I will wait testing this project then until things can work in VS. In my work setting it is a no-go if there is no (at least basic) VS support as I will never be able to convince colleges to use F# over C# otherwise.

**Comment by Alxandr** [comment]
Yeah. I get that. As said. You can get it to work in solutions that only have F# projects with a bit of manual setup at the start. But if you want C# projects to depend on those it becomes worse. Also, tooling support is entirely out of scope of this project. The point of this project is to get DNX to work with F#. VS plugins are completely out of my area of expertise.

**Comment by ghost** [comment]
I understand - but could the scope and limitations be stated in a readme or something? Otherwise some people may misunderstand what this project is about AND it may mean other developers are not working on VS plugins because they think this project is doing it already?
